PrebidMobileTests/RenderingTests/Tests/ParameterBuilderTests/PrebidParameterBuilderTest.swift

```diff
@@ -726,6 +725,55 @@ class PrebidParameterBuilderTest: XCTestCase {
         XCTAssert(bidRequest.extPrebid.targeting["includeformat"] as! Bool == true)
     }
 
+    func testIncludewinnersAndIncludeBidderKeysAreNil() {
+        //Default value
+        Prebid.shared.includeWinners = false
+        Prebid.shared.includeBidderKeys = false
+
+        let adUnit = BannerAdUnit(configId: "test", size: CGSize(width: 300, height: 250))
+        adUnit.adUnitConfig.adFormats = [.banner]
+        var bidRequest = buildBidRequest(with: adUnit.adUnitConfig)
+        XCTAssertNil(bidRequest.extPrebid.targeting["includewinners"])
+        XCTAssertNil(bidRequest.extPrebid.targeting["includebidderkeys"])
+    }
+
+    
+    func testIncludewinnersAndIncludeBidderKeysAreNotNil() {
+        //Default value
+        Prebid.shared.includeWinners = true
```

Comment:
@AbrahamArmasCordero , please, reset `Prebid.shared.includeWinners` and `Prebid.shared.includeBidderKeys` to default values in `reset` method inside `PBMSDKConfiguration+TestExtension.swift` file in order not to accidentally break other tests.

Reply:
sorry mb